netlify/edge-functions/og-image.tsx

```diff
@@ -0,0 +1,262 @@
+              flexDirection: 'row',
+              flexWrap: 'wrap',
+              maxHeight: 180, // Limit to 2 lines of authors
+              overflow: 'hidden',
+              paddingTop: -20,
+            }}
+          >
+            {withAuthors &&
+              parsedAuthors.slice(0, MAX_AUTHORS).map(({ name, github }) => {
+                return (
+                  <div
+                    style={{
+                      maxWidth: 1080,
+                      display: 'flex',
+                      flexDirection: 'row',
+                      flexWrap: 'wrap',
+                      marginBottom: 20,
+                      marginRight: 40,
+                    }}
+                  >
+                    <div
+                      style={{
+                        display: 'flex',
+                        width: 70,
+                        height: 70,
+                        borderRadius: '50%',
+                        background:
+                          'linear-gradient(135deg, #007FFF 0%, #FFFFFF 50%, #007FFF 100%)',
+                      }}
+                    >
+                      <img
+                        src={`https://github.com/${github}.png`}
+                        width={64}
+                        height={64}
+                        style={{ borderRadius: '50%', margin: 4 }}
+                        alt=""
+                      />
+                    </div>
+                    <div
+                      style={{
+                        marginLeft: 20,
+                        display: 'flex',
+                        flexDirection: 'column',
+                      }}
+                    >
+                      <span
+                        style={{
+                          fontSize: '28px',
+                          fontWeight: '600',
+                          lineHeight: '42px',
+                          textAlign: 'left',
+                          color: '#101418',
+                        }}
+                      >
+                        {name}
+                      </span>
+                      <span
+                        style={{
+                          fontSize: '20px',
+                          fontWeight: '500',
+                          lineHeight: '30px',
+                          textAlign: 'left',
+                          color: '#007FFF',
+                        }}
+                      >
+                        @{github}
+                      </span>
+                    </div>
+                  </div>
+                );
+              })}
+          </div>
+        </div>
+      </div>
+    ),
+    {
+      width: 1280,
+      height: 640,
+      // debug: true,
+      fonts: [
+        {
+          name: 'General Sans',
+          data: await fetch('https://fonts.cdnfonts.com/s/85793/GeneralSans-Medium.woff').then(
+            (a) => a.arrayBuffer(),
+          ),
+          weight: 1000,
+          style: 'normal',
+        },
```

Comment:
We don't seem to be using this weight here; maybe we can replace it for the IBM Plex Sans font so we can use it on the description text?

Reply:
Maybe also change the `name` there, right?